### PR TITLE
Crm 17275 import

### PR DESCRIPTION
**Pull request by eileenmcnaughton**

### CODE REVIEW

File: CRM/Contact/Import/Parser/Contact.php [hunk 1]
```diff
@@ -229,10 +235,7 @@ public function init() {
     }
 
     $this->_updateWithId = FALSE;
-    if (in_array('id', $this->_mapperKeys) || ($this->_externalIdentifierIndex >= 0 && in_array($this->_onDuplicate, array(
```
Comment (eileenmcnaughton) [inline CRM/Contact/Import/Parser/Contact.php]: In fact this change here is the only functional change. The rest is all tidy up & test support

**Comment by davecivicrm** [comment]
Test failures are unrelated.

**Comment by davecivicrm** [comment]
Reviewed and 'returned to development'. See my comment on the issue.

**Comment by eileenmcnaughton** [comment]
changed the labels as the only commit in this PR atm is cleanup

**Pull request merged**